app/src/main/assets/migrations/27.sql

```diff
@@ -1,3 +0,0 @@
-ALTER TABLE TransmitterData ADD COLUMN filtered_data DOUBLE;
```

Comment:
It is already added to 31.sql.  I decided it was easier to just keep the
two later migration files, and therefore 27.sql became redundant.
31 still keeps it as REAL rather than DOUBLE, but I thought that wouldn't
be a big issue.  Can change all REAL to DOUBLE if you prefer.

On Sun, May 24, 2015 at 12:15 PM, Stephen Black <notifications@github.com>
wrote:

> In app/src/main/assets/migrations/27.sql
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/30#discussion_r30949607>
> :
>
> > @@ -1,3 +0,0 @@
> > -ALTER TABLE TransmitterData ADD COLUMN filtered_data DOUBLE;
>
> This line does need to be in here I believe, we need to make sure at least
> one of these migrations adds filterd data to the TransmitterData table, I
> dont mind if you keep it in this file or add it into migration 32, either
> would be just fine!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/30/files#r30949607>
> .
>



-- 
John Stevens
"You are how you live, not what you have."